sonar-db/src/main/java/org/sonar/db/permission/template/PermissionTemplateDao.java

```diff
@@ -231,4 +231,17 @@ private Date now() {
   private static PermissionTemplateMapper mapper(DbSession session) {
     return session.getMapper(PermissionTemplateMapper.class);
   }
+
+  public void deleteByOrganization(DbSession dbSession, String organizationUuid) {
+    PermissionTemplateMapper templateMapper = mapper(dbSession);
+    PermissionTemplateCharacteristicMapper templateCharacteristicMapper = dbSession.getMapper(PermissionTemplateCharacteristicMapper.class);
+    List<Long> templateIds = templateMapper.selectTemplateIdsByOrganization(organizationUuid);
+    executeLargeInputsWithoutOutput(templateIds, (subList) -> {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the parentheses around the "subList" parameter [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1611)